samples/sm_host_shell/boards/nrf54l15dk_nrf54l15_cpuapp.overlay

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/ {
+	chosen {
+		ncs,sm-uart = &uart21;
+	};
+};
+
+/ {
+	aliases {
+		/delete-property/ led1;
+		/delete-property/ led3;
+		/delete-property/ sw1;
+		/delete-property/ sw2;
+	};
+};
+
+/delete-node/ &led1;
+/delete-node/ &led3;
+/delete-node/ &button1;
+/delete-node/ &button2;
```

Comment:
I'm a bit against of re-using GPIOs for UART that have anything other connected.

I would still propose to use the UART30, and document that it needs to be disconnected using the Board configurator.